doc/source/indexing.rst

```diff
@@ -78,8 +78,12 @@ of multi-axis indexing.
     *label* of the index. This use is **not** an integer position along the
     index)
   - A list or array of labels ``['a', 'b', 'c']``
-  - A slice object with labels ``'a':'f'``, (note that contrary to usual python
-    slices, **both** the start and the stop are included!)
+  - A slice object with labels ``'a':'f'``: note that contrary to usual python
+    slices, **both** the start and the stop are included, when present in the
+    index! If at least one of the two is absent but the index elements are
+    sorted (and can be compared against start and stop labels), then slicing
```

Comment:
as indicated in my comments 
pull these out to a separate note and put in the label section